utilities/src/main/java/io/syndesis/qe/utils/dballoc/DBAllocationManager.java

```diff
@@ -0,0 +1,95 @@
+package io.syndesis.qe.utils.dballoc;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.io.IOException;
+import java.io.Reader;
+import java.io.StringReader;
+import java.util.Map;
+import java.util.Properties;
+
+import io.syndesis.qe.utils.RestUtils;
+
+public class DBAllocationManager {
+
+    private static final String URL_GUI = "http://dballocator.mw.lab.eng.bos.redhat.com:8080/Allocator/AllocatorServlet";
```

Comment:
This URL need to be loaded from external config only. E.g. default to `localhost:8080` eventually.

Reply:
@dsimansk you mean from test.properties file?